bounties/pip/beam/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2021-01-08",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitrary Code Execution",
+    "Contributor": {
+      "Discloser": "Anon-Artist",
+      "Fixer": ""
+    },
+    "Package": {
+      "Registry": "pip",
+      "Name": "beam",
```

Comment:
Rename to: `https://pypi.org/project/apache-beam/`, both `Name` and the directory structure, thanks! 🍰